tools/Utilities/utils.ps1

```diff
@@ -0,0 +1,56 @@
+﻿# Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+
+<#
+       Install Required Tools
+#>
+Function Install-AzModule {
+    if (!(Get-Module -Name Az -ListAvailable)) {
+        Install-Module -Name Az -Repository PSGallery -Scope CurrentUser -Force -SkipPublisherCheck -AllowClobber 
+        Import-Module Az
+    }
+}
+<#
+    Connect to the Graph Tenant
+#>
+Function Connect-GraphTenant
+{
+    $defaultCertificate = Get-Certificate
```

Comment:
We should also document that one can substitute this line with their own cert if they are running the tests locally and don't have access to Azure KeyVault. This mostly applies to 3P contributors.

Reply:
I have simplified the process to use an in-memory certificate.